admin/js/wi-volunteer-management-block.js

```diff
@@ -6,6 +6,7 @@
 	const { Disabled, Toolbar, ToolbarButton }   = wp.components;
 	const { serverSideRender: ServerSideRender } = wp;
 	const { __ }                                 = wp.i18n;
+	
```

Comment:
Do you want to remove this blank line and the one just before defining the `isVolunteerManagementBlockRegistered` variable? If we aren't going to remove them, can we line up the equal signs to match up with their block of variables?

Reply:
Removed.